tasks/e2e/cypress/integration/01-tutorial/tutorial.spec.js

```diff
@@ -91,16 +108,37 @@ describe('The Redwood Tutorial - Golden path edition', () => {
   it('4. Getting Dynamic', () => {
     // https://redwoodjs.com/tutorial/getting-dynamic
     cy.writeFile(path.join(BASE_DIR, 'api/db/schema.prisma'), Step4_1_DbSchema)
-    cy.exec(`rm ${BASE_DIR}/api/db/dev.db`, { failOnNonZeroExit: false })
+    cy.wait(15000)
+    cy.task('execa', {
+      cmd: `rm ${BASE_DIR}/api/db/dev.db`,
```

Comment:
I get this error here 
![image](https://user-images.githubusercontent.com/30793/113834664-62df9400-978b-11eb-9d93-36f5c9641b34.png)

Reply:
Hmpf. Wtf!?

Reply:
Do you think watchers from the dev server do something different on windows?

Reply:
Yeah. I also can't use `rwt cpw` together with `yarn build:watch` because of file locks.

That said, this has actually passed a few times locally now 🤷